src/utils/event.ts

```diff
@@ -88,3 +88,20 @@ export const eventTimeComparator = (a: VenueEvent, b: VenueEvent) => {
 
   return a.duration_minutes - b.duration_minutes;
 };
+
+export const eventTimeAndOrderComparator = (a: VenueEvent, b: VenueEvent) => {
+  var aOrderPriority = a.orderPriority ?? 0;
+  var bOrderPriority = b.orderPriority ?? 0;
+  var aStartUtcSeconds = a.start_utc_seconds;
+  var bStartUtcSeconds = b.start_utc_seconds;
+
+  if (aOrderPriority === bOrderPriority) {
+    return aStartUtcSeconds < bStartUtcSeconds
+      ? -1
+      : aStartUtcSeconds > bStartUtcSeconds
+      ? 1
+      : 0;
+  } else {
+    return aOrderPriority > bOrderPriority ? -1 : 1;
+  }
```

Comment:
`var` -> `const`
`a < b ? -1 : 1` -> `a - b`
reuse `eventTimeComparator`

```suggestion
  const aOrderPriority = a.orderPriority ?? 0;
  const bOrderPriority = b.orderPriority ?? 0;

  if (aOrderPriority === bOrderPriority) {
    return eventTimeComparator(a, b);
  } else {
    return aOrderPriority - bOrderPriority;
  }
```

Reply:
fixed 6df0815